source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -1,26 +1,58 @@
 ---
-title: Ember 2.10.0 and 2.11.0-beta.1 Release
+title: Ember 2.10 and 2.11 Beta Released
 author: Godfrey Chan, Brendan McLoughlin, Nathan Hammond
 tags: Releases
 ---
 
+Today, the Ember project is releasing Ember.js, Ember Data and Ember CLI
+version 2.10.0.
+
+This also kicks off the 2.11 beta cycle for all sub-projects. We enourage our
+community (especially addon authors) to help test these beta builds and report
+any bugs before they are published as a final release in six weeks' time. (The
+[ember-try](https://github.com/ember-cli/ember-try) addon is a great way to do
+that.)
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
```

Comment:
I don't think all visitors need to be asked to let us know about the format. Seems a good thing to ask for feedback on in Slack after the fact.